Client/Frontend/Login/LoginListViewController.swift

```diff
@@ -209,28 +230,29 @@ extension LoginListViewController {
                 $0.selectionStyle = .none
                 $0.accessoryType = .disclosureIndicator
 
-                $0.setLines(loginInfo.hostname, detailText: loginInfo.username)
+                $0.setLines(loginInfo.displayURLString,
+                            detailText: loginInfo.usernameValue)
                 $0.imageView?.contentMode = .scaleAspectFit
                 $0.imageView?.image = FaviconFetcher.defaultFaviconImage
                 $0.imageView?.layer.borderColor = BraveUX.faviconBorderColor.cgColor
                 $0.imageView?.layer.borderWidth = BraveUX.faviconBorderWidth
                 $0.imageView?.layer.cornerRadius = 6
                 $0.imageView?.layer.cornerCurve = .continuous
                 $0.imageView?.layer.masksToBounds = true
-            }
-                      
-            if let loginHostnameURL = URL(string: loginInfo.hostname) {
-                let domain = Domain.getOrCreate(forUrl: loginHostnameURL, persistent: true)
                 
-                cell.imageView?.loadFavicon(
-                    for: loginHostnameURL,
-                    domain: domain,
-                    fallbackMonogramCharacter: loginHostnameURL.baseDomain?.first,
-                    shouldClearMonogramFavIcon: false,
-                    cachedOnly: true)
-            } else {
-                cell.imageView?.clearMonogramFavicon()
-                cell.imageView?.image = FaviconFetcher.defaultFaviconImage
+                if let signOnRealmURL = URL(string: loginInfo.signOnRealm) {
+                    let domain = Domain.getOrCreate(forUrl: signOnRealmURL, persistent: true)
+    
+                    cell.imageView?.loadFavicon(
```

Comment:
We use our `loadFavicon` method from `FavIconFetcher` here( this method is one we use in entire application (tabs, history, bookmarks etc). this method acts caches the favicons for websites if they are fetched properly from that domain. They are trying to be fetched again in case they are failed and a monogram is used instead. 

In addition while traversing the table list, the cell is reused so entire cell doesnt do an operation as long as you are in that particular screen.